.github/workflows/archived/backend-tests.yml

```
@@ -135,7 +135,7 @@ jobs:
         env:
           ASPNETCORE_ENVIRONMENT: Testing
           CONNECTION_STRING: "Host=localhost;Port=5432;Database=terrafusion_test;Username=postgres;Password=postgres"
-          REDIS_CONNECTION: "localhost:6379"
+          REDIS_CONNECTION: "localhost:\${{TF_REDIS_PORT:-6379}}"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 138: found unknown escape character

(syntax-check)